threeML/plugins/OGIP/likelihood_functions.py

```diff
@@ -114,4 +114,111 @@ def poisson_observed_gaussian_background(observed_counts, background_counts, bac
     log_likes[nidx] = observed_counts[nidx] * np.log(expected_model_counts[nidx] + 1e-100) - \
                       expected_model_counts[nidx] - logfactorial(observed_counts[nidx])
 
-    return np.sum(log_likes)
\ No newline at end of file
+    return np.sum(log_likes)
+
+
+# For some statistical functions (WAIC) it is desirable to look at the likelihood of each individual data point.
+# These functions return the vector of the observation for that purpose
+
+def poisson_log_likelihood_ideal_bkg_vector(observed_counts, expected_bkg_counts, expected_model_counts):
```

Comment:
This must have been some leftover from very old things. I don't think I touched this.